<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSectionKeyboard</class>
 <widget class="QWidget" name="ConfigSectionKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="TableKey" name="tableKeyboardMap">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>65</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <row>
      <property name="text">
       <string>Octave #1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Octave #2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Octave #3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Octave #4</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string notr="true">Do</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Do#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Ré</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Ré#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Mi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Fa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Fa#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Sol</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Sol#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">La</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">La#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Si</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Do</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>First C</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboDo">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string notr="true">0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">36</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">48</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">60</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">72</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">84</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Default velocity</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinDefaultVelocity">
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>127</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableKey</class>
   <extends>QTableWidget</extends>
   <header>tablekey.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
